docs/UserGuide.md

```diff
@@ -189,24 +198,36 @@ Format: `apply INDEX n/NAME d/DESCRIPTION [as/APPLICATION_STATUS]`
 
 * Adds an application record for the company at the specified `INDEX`. The index refers to the index number shown in the displayed company list. The index **must be a positive integer** 1, 2, 3, …​
 * `APPLICATION_STATUS` can only take the values `APPLIED`, `OA`, `INTERVIEWED`, `OFFERED`, `ACCEPTED`, `REJECTED`
-and will take be `APPLIED` if not specified.
+and will be `APPLIED` if not specified.
 
 <box type="tip" seamless>
 
-**tip:** applying to a company automatically changes the company's status to `applied`.
+**Tip:** applying to a company automatically changes the company's status to `applied`.
 </box>
 
 Examples:
 * `apply 1 n/Software Engineer Intern d/Requires knowledge of ReactJS and ExpressJS`
 * `apply 2 n/Product Management Intern d/Requires Figma as/OA`
 * `apply 3 n/Devops Engineer Intern d/Requires knowledge in networks as/OFFERED`
 
+--------------------------------------------------------------------------------------------------------------------
+
+### Listing all companies : `list`
+
+Shows a list of all companies in the address book.
+
+Format: `list`
+
+--------------------------------------------------------------------------------------------------------------------
+
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+<span style="color:red">**Deletes all entries**</span> from the address book.
```

Comment:
Nice that you made this read. 👍 
Another enhancement could be to add a warning box that clearing permanently removes data.